Examples/SpatialObjects/SpatialObjectTransforms.cxx

```diff
@@ -19,17 +19,55 @@
 // Software Guide : BeginLatex
 //
 // \index{itk::SpatialObjectTransforms} This example describes the different
-// transformations associated with a spatial object.
-//
-//
-// \begin{figure} \center
-// \includegraphics[width=0.9\textwidth]{SpatialObjectTransforms}
-// \itkcaption[SpatialObject Transformations]{Set of transformations associated
-// with a Spatial Object}
-// \label{fig:SpatialObjectTransforms}
-// \end{figure}
-//
-// Figure~\ref{fig:SpatialObjectTransforms} shows our set of transformations.
+// transformations and "spaces" ssociated with a spatial object.
```

Comment:
Great documentation!

Reply:
The 'a' from associated was removed.

Reply:
I think explicitly writing World space and Object space would be more explicit than just "spaces".

Reply:
Fixed